src/components/Label/index.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from '@emotion/styled'
+import React, { FunctionComponent } from 'react'
+
+const StyledLabel = styled.label`
+  color: ${({ theme }) => theme.colors.gray950};
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 500;
+  margin-bottom: 8px;
+`
+
+const Label: FunctionComponent = props => <StyledLabel {...props} />
```

Comment:
```suggestion
const Label: FunctionComponent<LabelHTMLAttributes<HTMLLabelElement>> = props => <StyledLabel {...props} />
```

Reply:
ffs, that component proxy really stinks ...